SQL/02_Tabloid_Seed_Data.sql

```diff
@@ -17,16 +17,16 @@ values (1, 'C#'), (2, 'JavaScript'), (3, 'Funny'), (4, 'Scary');
 set identity_insert [Tag] off
```

Comment:
Please add this file to your .gitignore

Reply:
Do I just type the name of the file in the .gitignore?